src/Servant/To/Elm.hs

```diff
@@ -625,3 +482,255 @@ instance HasElmType (Servant.MultipartData tag) where
 instance HasElmEncoder (Servant.MultipartData tag) (Servant.MultipartData tag) where
   elmEncoder =
     "Basics.identity"
+
+-------------------------------------------------------------------------------
```

Comment:
From here down it's just definitions that used to be in the `where` of `elmEndpointDefinition`, but which I needed to pull to the top level so that they could also be used by `elmEndpointRequestInfo`. Let me know if you would like these hidden via explicit exports, placed in a new internal module, etc.

Reply:
Cool! An explicit export list would be preferrable I think. I doubt any of these are useful on their own.

Reply:
Done!

Reply:
Cheers!